Jamulus.pro

```diff
@@ -1,5 +1,11 @@
 VERSION = 3.10.0dev
 
+# Using lrelease and embed_translations only works for Qt 5.12 or later.
+# See https://github.com/jamulussoftware/jamulus/pull/3288 for these changes.
+lessThan(QT_MAJOR_VERSION, 5) | equals(QT_MAJOR_VERSION, 5) : lessThan(QT_MINOR_VERSION, 12) {
+    error(Jamulus requires at least Qt5.12)
```

Comment:
```suggestion
    error(Jamulus requires at least Qt5.12. Check PR #3288)
```

Reply:
I'd suggest the full URL in case the source ever moves off GitHub (the PR reference alone could become meaningless).

Reply:
Done, see 97306e7